src/components/Box.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+function Box({ size , children}) {
```

Comment:
`const { theme } = useContext(ThemeContext);` should be used here to control the background